bin/install-wordpress.sh

```diff
@@ -87,6 +87,10 @@ if [ "$CURRENT_URL" != "http://localhost:$HOST_PORT" ]; then
 	docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run --rm -u 33 $CLI option update siteurl "http://localhost:$HOST_PORT" --quiet
 fi
 
+# Configure site constants.
+echo -e $(status_message "Configuring site constants...")
+docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run --rm -u 33 $CLI config set WP_DEBUG true --quiet
```

Comment:
Do we want to run the e2e tests in debug mode?

Reply:
It's risky but we can give it a try and see if it doesn't make the experience harder.

By the way, it would solve the issue @iseulde was trying to tackle earlier. Maybe we could put it behind some flag? It seems to be very useful when developing locally.

Reply:
See #12457 for reference.

Reply:
I now see that it's different `SCRIPT_DEBUG` vs `WP_DEBUG`. I think I should finish my work for today 😅

Reply:
I don't know what would be risky about it. I run with `WP_DEBUG` constantly in my development environment, exactly to be able to catch an issue like this one.

Reply:
My question was:

Could a debug environment be different from a regular prod environment we should mimic in tests.

I don't have objections though.